app/javascript/application.js

```diff
@@ -0,0 +1,33 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import "@hotwired/turbo-rails"
```

Comment:
I'm rather unfamiliar with JavaScript, so I'll probably skip any review of that and html/css, hope that's okay. Will stick to Ruby for now.